storage-proofs/core/src/parameter_cache.rs

```diff
@@ -25,7 +25,7 @@ pub const PARAMETER_METADATA_EXT: &str = "meta";
 pub const VERIFYING_KEY_EXT: &str = "vk";
 
 #[derive(Debug)]
-struct LockedFile(File);
+pub struct LockedFile(File);
```

Comment:
This might belong in a more general place if being used elsewhere now, too. Let's consider moving later.